.github/workflows/push.yml

```diff
@@ -34,6 +34,7 @@ env:
   COMPOSE_HTTP_TIMEOUT: 180
   GITHUB_ACTIONS_CACHE_URL: https://cache.dev01.devland.is/
   SKIP_GENERATED_CACHE: ${{ contains(github.event.pull_request.labels.*.name, 'skip-generated-cache') }}
+  NX_AFFECTED_ALL: ${{ contains(github.event.pull_request.labels.*.name, 'nx-affect-all') }}
```

Comment:
This won't have any effect on `main`, since it isn't a PR and doesn't have a PR event (possibly failing here).